src/systemcmds/tests/test_mathlib.cpp

```diff
@@ -371,6 +372,24 @@ bool MathlibTest::testQuaternionRotate()
 	return true;
 }
 
+bool MathlibTest::testFinite()
+{
+	ut_assert("PX4_ISFINITE(0.0f)", PX4_ISFINITE(0.0f) == true);
+	ut_assert("PX4_ISFINITE(-0.0f)", PX4_ISFINITE(-0.0f) == true);
+	ut_assert("PX4_ISFINITE(1.0f)", PX4_ISFINITE(1.0f) == true);
+	ut_assert("PX4_ISFINITE(-1.0f)", PX4_ISFINITE(-1.0f) == true);
+
+	ut_assert("PX4_ISFINITE(NAN)", PX4_ISFINITE(NAN) == false);
+	ut_assert("PX4_ISFINITE(1/0)", PX4_ISFINITE(1.0f / 0.0f) == false);
+	ut_assert("PX4_ISFINITE(0/0)", PX4_ISFINITE(0.0f / 0.0f) == false);
+	ut_assert("PX4_ISFINITE(INFINITY)", PX4_ISFINITE(INFINITY) == false);
+	ut_assert("PX4_ISFINITE(NAN * INFINITY)", PX4_ISFINITE(INFINITY) == false);
+	ut_assert("PX4_ISFINITE(NAN * 1.0f)", PX4_ISFINITE(INFINITY) == false);
+	ut_assert("PX4_ISFINITE(INFINITY * 2.0f)", PX4_ISFINITE(INFINITY) == false);
```

Comment:
For some of these you forgot to update the assertion.

Reply:
Thanks.